test/units/testsuite-50.mountnfsd.sh

```diff
@@ -49,6 +49,15 @@ systemd-dissect --image-policy='root=unprotected:=absent+unused' --mtree /var/tm
 (! runas testuser systemd-dissect /var/tmp/unpriv.raw)
 (! runas testuser systemd-dissect --mtree /var/tmp/unpriv.raw)
 
+if ( SYSTEMD_LOG_TARGET=console varlinkctl call \
+        /run/systemd/userdb/io.systemd.NamespaceResource \
+        io.systemd.NamespaceResource.AllocateUserRange \
+        '{"name":"test-supported","size":65536,"userNamespaceFileDescriptor":0}' 2>&1 || true ) |
+            grep -q "io.systemd.NamespaceResource.UserNamespaceInterfaceNotSupported"; then
```

Comment:
The subshell seems unnecessary here, just do:

```suggestion
if SYSTEMD_LOG_TARGET=console varlinkctl call \
        /run/systemd/userdb/io.systemd.NamespaceResource \
        io.systemd.NamespaceResource.AllocateUserRange \
        '{"name":"test-supported","size":65536,"userNamespaceFileDescriptor":0}' |& \
            grep -q "io.systemd.NamespaceResource.UserNamespaceInterfaceNotSupported"; then
```

Reply:
@mrc0mmand Does this handle pipefail? That's why I did the subshell

Reply:
If the varlink call fails the condition won't be true and we won't skip the test, so it should be what we want?

Reply:
@mrc0mmand We expect the varlink call to fail, and then we check that the error is the one on which we want to skip.

Reply:
(I stole this pattern from `testcase_bootctl_varlink`)

Reply:
Ah, ignore me then, just drop the whitespaces before ().